kyuubi-server/src/main/antlr4/org/apache/kyuubi/sql/KyuubiTrinoFeBaseParser.g4

```diff
@@ -54,9 +54,19 @@ statement
       CAST LEFT_PAREN NULL AS SMALLINT RIGHT_PAREN KEY_SEQ COMMA
       CAST LEFT_PAREN NULL AS VARCHAR RIGHT_PAREN PK_NAME
       WHERE FALSE                                                                                       #getPrimaryKeys
+    | EXECUTE IDENTIFIER (USING parameterList)?                                                         #getPrepareParameters
+    | PREPARE IDENTIFIER FROM statement                                                                 #getPrepareSql
```

Comment:
let's follow the Trino style:
`getPrepareParameters` -> `execute`
`getPrepareSql` -> `prepare`

Reply:
ok，I misunderstood the api named mapping relationship. It should correspond to the engine's api

Reply:
> please remove test/debug code when it's ready to review, thank you

Thanks for your reminder and  I will close it this weekend . Now this g4 file cannot match all cases， are there any matching rules that can be reused to parse trino's execute-grammar

Reply:
is that required ? if so can we reuse that from Trino ? introduce dependency or copy some code.